src/java/com/vaklinov/zcashui/ZCashUI.java

```diff
@@ -974,6 +984,12 @@ public static void possiblyCreateZelCashConfigFile() throws IOException {
 			configOut.println("");
 			configOut.println("rpcallowip=127.0.0.1");
 			configOut.println("server=1");
+			configOut.println("daemon=1");
+			configOut.println("txindex=1");
+			configOut.println("logtimestamps=1");
```

Comment:
We do not want to have experimental features in production releases. Native encryption in zelcash is still an experimental feature and should be treated as such.
Please remove these lines:
```
			configOut.println("experimentalfeatures=1");
			configOut.println("developerencryptwallet=1");
```
These lines
```
			configOut.println("daemon=1");
			configOut.println("logtimestamps=1");
			configOut.println("maxconnections=256");
```
are also not needed for running zelnodes. 
I agree with adding txindex=1